Correctly handle percentage widths for table cells

ad-si opened the pull request:
Fixes https://github.com/privateOmega/html-to-docx/issues/189

ad-si committed:
style: use .js for require to avoid issues when executing with Node.js

ad-si committed:
chore: use import instead of require in Node.js example

ad-si committed:
fix: correctly handle a percentage width for table cells

nicolasiscoding commented:
This repo isn't being maintained anymore, but if you go to the TurboDocx/html-to-docx one we're happy to take a look if we haven't fixed this already in our hard fork

ad-si committed:
chore: commit built files for easy installation via npm

ad-si committed:
fix: remove prepare script so repo can be directly used as dependency

ad-si commented:
Thanks for the heads up @nicolasiscoding. However, this version here seems to have quite a few more commits. Can you say something about the differences / improvements in your fork?

nicolasiscoding commented:
@ad-si i haven't seen what has been merged lately in this repo as we stopped contributing about 2 years ago. 

In our fork, we actively maintain it and continually deploy with features and bug fixes. AFAIK from when we began the fork we support more tags and edge cases with complex OOXML, better error handling, and handle most document editors, etc.

ad-si committed:
Upgrade to latest version of ESLint and fix emerging errors

ad-si committed:
Only build esm version, upgrade all dependencies

ad-si committed:
Move full HTML example to tests

ad-si committed:
Use string manipulation instead of a regex for extracting base64 data

This should avoid exceeding the maximum call stack size as described at
https://stackoverflow.com/q/63352844

ad-si committed:
Fix typo

ad-si committed:
CI: Add workflow that executes the tests

ad-si commented:
Thank you for the infos @nicolasiscoding!
We decided to maintain our own fork at https://github.com/OxfordAbstracts/html-to-docx. We converted the repo to TypeScript, removed obsolete stuff, and made it more robust for big HTML files.
